### PR TITLE
Python API v2 examples

### PR DESCRIPTION
**Pull request by KrzysztofTucholski**

### CODE REVIEW

File: python/2.0/offers-get.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+API_KEY = open('./api_key').read()
+assert API_KEY is not None
+
+API_BASE = 'https://api.walutomat.pl'
+URI = '/api/v2.0.0/market_fx/orders/active'
+
+private_key = crypto.load_privatekey(crypto.FILETYPE_PEM, open('./private.key').read())
+
+ts = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%SZ")
+dataToSign = f'{ts}{URI}'
```
Comment (sicarrots) [inline python/2.0/offers-get.py]: w pythonie używamy snake_case :)

File: python/2.0/offers-get.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
```
Comment (suside) [inline python/2.0/offers-get.py]: I'm confused a little by that directory naming: `1.0` + `2.0` it kinda looks like python versions.
How about `APIv1` + `APIv2` wouldn't that be more intuitive?

Comment (suside) [reply python/2.0/offers-get.py]: And python path is mentioned in `README.md`
https://github.com/Currency-One/api-examples/blob/98d3c9283fd0a8055992f981e30d579cdcfeb463/README.md#L20

File: python/2.0/offers-get.py (hunk 1 above)
Comment (KrzysztofTucholski) [reply python/2.0/offers-get.py]: a sorry

File: python/2.0/offers-get.py (hunk 2 above)
Comment (KrzysztofTucholski) [reply python/2.0/offers-get.py]: thx

**Pull request merged**